crates/pathfinder/src/sequencer/reply.rs

```diff
@@ -124,6 +124,21 @@ pub enum Status {
     Aborted,
 }
 
+impl std::fmt::Display for Status {
```

Comment:
Hmm this kinda calls for `derive_more`, if we can accept another dependency...

Reply:
Thought of that but didn't want to add a new dependency just for this single case.